### DISCUSSION TITLE
You removed getArguments in SelectionSet in 16.1, how should i change my code?

### DISCUSSION THREAD

**Discussion by knotenpunkt** [graphql-java/graphql-java#2447] [Q&A] [answered]
query-example
```
 getFoo {
    bar
    randomExample(count: 3) {
      blub
    }
  }
```
code example in java

`Long count = (Long) (dfe.getSelectionSet().getArguments().get("randomExample").get("count"));`

That code worked on version 15.0 but not anymore in 16.1, because you removed getArguments on the SelectionSet.
So how i can achieve the same not having to iterate through lists?

greetings knotenpunkt

**Comment by dfa1** [comment, marked as answer]
perhaps something like:

`Long count = (Long) (dfe.getFieldsGroupedByResultKey().get("randomExample").get(0).getArguments().get("count")

You may want to dig a bit here: afaa9aa1f321ef4aed7ef6ccac3e2efdd8413874.
